utils/run_test.sh

```diff
@@ -131,8 +131,10 @@ if [ -d "/mnt/daos" ]; then
     run_test "${SL_PREFIX}/bin/vos_size"
     run_test "${SL_PREFIX}/bin/vos_size.py" \
              "${SL_PREFIX}/etc/vos_size_input.yaml"
+    run_test "${SL_PREFIX}/bin/vos_size_dfs_sample"
```

Comment:
I wonder if we shouldn't call this as part of the build to generate vos_dfs_sample.yaml and place it in /etc where it was before?

Reply:
Done, I was figuring out how to do it. The build system is a bit tricky.
Unfortunately, I messed up this branch and now there are other files from others commits that don't belong to my PR.
So, I created 2947. I guess I am going to close this PR